tools/migrator/src/migrations/consolidate-props/testing.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
I'm a bit confused what this file is for

Reply:
For testing purposes I added this file to test that running `yarn lg migrator` works in the project. If you run `yarn lg migrator consolidate-props 'tools/migrator/src/migrations/consolidate-props/testing.tsx'`, it will update `testing.tsx`.

Reply:
What's the difference between `/testing.tsx` and the`/*.input.tsx`/`/*.output.tsx` files?

Reply:
`/testing.tsx` is tested by running `yarn lg migrator consolidate-props` and `/*.input.tsx/`/`*.output.tsx` is tested with jest using,  `yarn test /tools/codemods`

Reply:
I'm not sure these files should stay then, since they're only for manual testing. The only files used in automated tests are the `/*.input.tsx/`/`*.output.tsx` files

Reply:
i'll remove them